.github/workflows/cd.yaml

```diff
@@ -5,9 +5,14 @@ on:
     tags:
       - '*'
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
@amessing-bdai does this apply for tag push events?

Reply:
I believe `github.event.pull_request.number` is for PR and `github.ref` is for push

Reply:
Ah, I missed the `||` on first ready. Nevermind.